tests/test_shtab.py

```diff
@@ -108,6 +108,8 @@ def test_prog_scripts(shell, caplog, capsys):
             "#compdef script.py",
             "_describe 'script.py commands' _commands",
         ]
+    elif shell == "tcsh":
+        assert script_py == ["complete script.py \\"]
```

Comment:
:sweat_smile: doesn't look like it's doing anything?

Reply:
Ah, now I understand what you mean. As you can see on the top comment, I generate multiple lines separated by `\\`, the line continuation. That's why it seems it's not doing anything. :grin: 

I could put it all in a giant line, but that hurts readability, one of the (good) ideas behind the project.

Reply:
sure I guess the tests are fairly basic anyway :)